src/SFA.DAS.EmployerFinance/Services/DasForecastingService.cs

```diff
@@ -74,5 +75,57 @@ public async Task<ExpiringAccountFunds> GetExpiringAccountFunds(long accountId)
                 return null;
             }
         }
+
+        public async Task<ProjectedCalculation> GetProjectedCalculations(long accountId)
+        {
+            _logger.Info($"Getting projected calculations for account ID: {accountId}");
+
+            var expiredFundsUrl = $"/api/accounts/{accountId}/AccountProjection/projected-summary";
+
+            var accessToken = await _azureAdAuthService.GetAuthenticationResult(
```

Comment:
Could we reuse common code between this method and the Expired funds one? They are using the same api

Reply:
Done